package.json

```diff
@@ -106,7 +106,7 @@
     "brace": "0.11.1",
     "bunyan": "1.7.1",
     "cache-loader": "1.0.3",
-    "chalk": "2.3.0",
+    "chalk": "^2.3.2",
```

Comment:
thoughts on ^2.4.1? i don't have a preference but thought i'd note it

Reply:
@jbudz looking at the changelog it seems okay to go with ^2.4.1 😃